modules/rosa-cluster-logging-collector-log-forward-sts-cloudwatch.adoc

```diff
@@ -0,0 +1,195 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+:_content-type: PROCEDURE
+
+[id="cluster-logging-collector-log-forward-sts-cloudwatch_{context}"]
+= Forwarding logs to Amazon CloudWatch from STS enabled clusters
+
+For clusters with AWS Security Token Service (STS) enabled, create the AWS IAM roles and policies that will allow the log forwarding, and a `ClusterLogForwarder` custom resource (CR) with an output for CloudWatch.
+
+[NOTE]
+====
+This feature is not supported by the vector collector.
+====
+
+.Prerequisites
+
+* {logging-title-uc}: 5.5 and later
+
+.Procedure
+. Prepare the AWS account:
+.. Create an IAM policy JSON file with the following content:
++
+[source,json]
+----
+{
+"Version": "2012-10-17",
+"Statement": [
+    {
+    "Effect": "Allow",
+    "Action": [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:DescribeLogGroups",
+      "logs:DescribeLogStreams",
+      "logs:PutLogEvents",
+      "logs:PutRetentionPolicy"
+    ],
+    "Resource": "arn:aws:logs:*:*:*"
+    }
+  ]
+}
+----
++
+.. Create an IAM trust JSON file with the following content:
++
+[source,json]
+----
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Federated": "arn:aws:iam::<you_aws_account_id>:oidc-provider/<openshift_oidc_provider>" <1>
+      },
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {
+        "StringEquals": {
+          "<openshift_oidc_provider>:sub": "system:serviceaccount:openshift-logging:logcollector" <2>
+        }
+      }
+    }
+  ]
+}
+----
++
+--
+<1> Specify your AWS account ID and the OpenShift OIDC provider endpoint. Obtain the endpoint by running the the following command:
++
+[source,terminal]
+----
+$ rosa describe cluster \
```

Comment:
Waiting for confirmation about this command, since it didn't seem to work as we expected. Oren Kashi was after a command that could work on hosted control planes too.

Reply:
@0kashi Following up - is this command appropriate?

Reply:
This command will work, but I think that for docs we might simplify it.

Reply:
In light of not being able to use `jq` lets use this command: `rosa describe cluster -c <cluster name> -o yaml | awk '/oidc_endpoint_url/ {print $2}' | cut -d '/' -f 3,4`